cmake/packaging.cmake

```
@@ -160,4 +163,15 @@ cpack_add_component(
 )
 cpack_add_component(ttnn-validation GROUP nn-validation)
 
+cpack_add_component_group(ttml)
+cpack_add_component(
+    ttml
+    DEPENDS
+        nn
+        metalium
+    GROUP ttml
+    DESCRIPTION "TT-Train runtime library"
+)
```

I assume you wouldn't want a package named `tt-ttml`.  Since `tt-` is automatically prefixed, we just use the stuff after that as our component name.

```suggestion
cpack_add_component_group(ml)
cpack_add_component(
    ttml
    DEPENDS
        nn
        metalium
    GROUP ml
    DESCRIPTION "TT-Train runtime library"
)
```